frontend/src/routes/Infrastructure/ClusterTemplates/ClusterTemplatesPage/ClusterTemplatesTab.tsx

```diff
@@ -137,33 +158,35 @@ const ClusterTemplatesTab = () => {
   const [templates, loaded, loadError] = useClusterTemplates();
 
   return (
-    <PageSection>
-      <TableLoader
-        loaded={loaded}
-        error={loadError}
-        errorId="templates-load-error"
-        errorMessage={t('The cluster templates could not be loaded.')}
-      >
-        <TableComposable
-          aria-label="Cluster templates table"
-          id="cluster-templates-table"
-          variant="compact"
+    <Page>
+      <PageSection>
+        <TableLoader
```

Comment:
The content of the page section needs to be wrapped in a `<Card/>` so the table, loader and error state get correct shadow as shown here: https://www.patternfly.org/v4/components/table/react-demos/compact/ 
The TableLoader will need some spacing inside the Card, so use also <CardBody> as an optional wrapper prop in TableLoader to wrap error and skeletons.